### ISSUE TITLE
More deprecation warnings with python 3.7 (cut'n'paste)

### ISSUE THREAD

**Issue by 7eggert** [kepstin/regainer#2]
```$ ./regainer foo.mp3
Namespace(FILE=['foo.mp3'], album=deque([]), dry_run=False, exclude=deque([]), force=False, jobs=2, track=deque([]))
./regainer:725: DeprecationWarning: 'with (yield from lock)' is deprecated use 'async with lock' instead
  with (yield from self.job_sem):
./regainer:732: DeprecationWarning: 'with (yield from lock)' is deprecated use 'async with lock' instead
  with (yield from self.job_sem):
./regainer:738: DeprecationWarning: 'with (yield from lock)' is deprecated use 'async with lock' instead
  with (yield from self.job_sem):

foo.mp3
Track: I: -17.90 LUFS, Peak: -5.80 dBFS; Album: I: None, Peak: None
Rescanned loudness
Updated tags
```

**Comment by kepstin** [comment]
Fixed in b6e0dbd499ddd96ed25100f601fa34743b55b6d1

**Comment by 7eggert** [comment]
tested

reding the diff, I found the comments about installing. I can recommend the command `install -T regainer.py /usr/local/bin/regainer`, as it will correctly set permissions and owner (having user-writable files in global bin directories is a BAD idea).
